imports/components/@primitives/UI/card-slider/SliderCard.js

```diff
@@ -1,11 +1,13 @@
 // @flow
 
 type ISliderCard = {
```

Comment:
@richarddubay I should have caught this earlier on. This isn't really a generic enough slider card. Can we rename it to MetricCard?